torchtune/generation/_generation.py

```diff
@@ -366,7 +366,7 @@ def generate(
         tokens, logits = custom_generate_next_token(
             model,
             input_pos=curr_input_pos,
-            x=tokens,
+            x=tokens.clone(),
```

Comment:
This is needed as cudagraphs is complaining about tensors being overwritten from previous graphs.